boards/st/stm32h750b_dk/Kconfig.stm32h750b_dk

```diff
@@ -3,3 +3,12 @@
 
 config BOARD_STM32H750B_DK
 	select SOC_STM32H750XX
+
+if DISPLAY
+
+# MEMC needs to be enabled in order to store
+# display buffer to external SDRAM connected to FMC
+config MEMC
+	default y
+
+endif # DISPLAY
```

Comment:
Already included in `boards/arm/stm32h750b_dk/Kconfig.defconfig` doesn't look required here.